### PR TITLE
this PR adds section for explaining stack choice of talawa (addresses the previous fix for showing changes to 5k+ files )

### LINKED ISSUE #450: Feature Request: Add to "Applications" section
**Is your feature request related to a problem? Please describe.**
Our applications page is missing detail on why we chose typescript, flutter, and monogodb. 

**Describe the solution you'd like**
Add to the "Applications" page information on why we used typescript, flutter, and monogodb. Our youtube page has videos that could be a helpful resource: https://www.youtube.com/@PalisadoesOrganization/videos. 

**Describe alternatives you've considered**
N/A

**Approach to be followed (optional)**
N/A

**Additional context**
N/A

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by SalilHarit**
What kind of change does this PR introduce?

Added to the "Applications" page information on why talawa uses typescript, flutter, and monogodb in their projects

Issue Number:

Fixes #450 

Did you add tests for your changes?

No

Snapshots/Videos:

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

no
**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-docs/blob/develop/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

File: docs/applications/architecture.md [hunk 1]
```diff
@@ -30,4 +30,29 @@ This diagram outlines how the three Talawa components interact
 
 ![img](/img/Talawa.jpg)
 
+### Choosing the Right Stack
+
+Why is it Important ?
```
Comment (Cioppolo14) [inline docs/applications/architecture.md:35]: can you change the I in Important to lowercase (i)?

File: docs/applications/architecture.md [hunk 2]
```diff
@@ -30,4 +30,29 @@ This diagram outlines how the three Talawa components interact
...
+
+Overall, choosing the right technology stack for a web development project can help ensure that the project meets its requirements, is developed quickly and efficiently, is scalable, well-supported, and affordable
```
Comment (Cioppolo14) [inline docs/applications/architecture.md]: Add a . after affordable.

**Review by Cioppolo14 — CHANGES_REQUESTED** [review]
Can you discuss in the sections why Talawa specifically these?

**Comment by SalilHarit** [comment]
> Can you discuss in the sections why Talawa specifically these?

sure ! i've had a doubt for quite a while which prevented me from going specific like when we talk about talawa in the terms of used technologies in their project does it mean all of their projects like api ,mobile app and web portal or just mobile app

**Comment by SalilHarit** [comment]
> Can you discuss in the sections why Talawa specifically these?

i've discussed where different technologies are used in different places using examples is it enough ?

**Review by Cioppolo14 — APPROVED** [review]

**Pull request merged**